src/Chainweb/Payload/PayloadStore/HashMap.hs

```diff
@@ -0,0 +1,64 @@
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+-- |
+-- Module: Chainweb.Payload.PayloadStore.HashMap
+-- Copyright: Copyright © 2019 Kadena LLC.
+-- License: MIT
+-- Maintainer: Lars Kuhtz <lars@kadena.io>
+-- Stability: experimental
+--
+-- TODO
```

Comment:
With this PR this is only used in testing. I don't care about the name, so happy to rename it.

Reply:
A related question is if we want to use `RocksDB` everywhere in the tests. (It seems to perform fine.) If not, we may consider providing an in-memory implementation of the API in `Data.CAS.RocksDB instead of using this module.